src/QsFmt/Formatter/SyntaxTree/Statement.fsi

```diff
@@ -255,36 +275,160 @@ and internal For =
         Block: Statement Block
     }
 
+/// <summary>
+/// The concluding section of an <c>until</c> statement.
+/// </summary>
+and internal UntilCoda =
```

Comment:
What's a coda? I don't think I've heard that before.

Reply:
The definition for coda is "a concluding event, remark, or section". I found it was a good description to describe the end-part of the statement.